### ISSUE TITLE
News

### ISSUE THREAD

**Issue by Hockeyfan360** [Superior-Development/IcarusMod#135]
I recently pushed a settings table to MySQl. I did not include a footer, because I thought we could put news there. Since that is postponed, where else could we use news? Send a chat message when they join? Auto broadcast the news every 5m or so?

**Comment by Hockeyfan360** [comment]
@Superior-Development/superior-freedom-network-pull-push Thoughts?

**Comment by Wild1145** [comment]
I dont think we _need_ any sort of server side news system, the tab footer would be the best place for it, anything that important would go on the forums...

**Comment by Scuph** [comment]
@Wild1145 I know this is off-topic, but if you can PM me your skype on the TF forums, I have stuff we should talk about.

**Comment by Wild1145** [comment]
@Scuph Send me an e-mail or something like that, I dont read the TF forums anymore.

**Comment by Hockeyfan360** [comment]
Do you think it would be worth updating every so often?

**Comment by Wild1145** [comment]
If its stored in the MySQL Database we can update it manually when it is so needed...
